vizier/backend/src/info/vizierdb/commands/plot/PlotUtils.scala

```diff
@@ -118,13 +163,14 @@ object PlotUtils
       )
     }
 
-
     PlotUtils.Series(
       dataset = datasetName,
       x = dataframe.columns(xIndex),
       y = dataframe.columns(yIndex),
       dataframe = dataframe,
       regression = regression,
+      sort = sort,
```

Comment:
`sort` is a pre-processing step on the dataframe.  The Series object itself shouldn't care whether the data is sorted or not.  Remove this field please.

Reply:
not yet resolved